mucommander-core/src/main/java/com/mucommander/search/LastSearchQuery.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of muCommander, http://www.mucommander.com
+ *
+ * muCommander is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * muCommander is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.mucommander.search;
+
+/**
+ * A class to keep the last search query and its options.
+ */
+public final class LastSearchQuery {
```

Comment:
why do we need this class, can't we use the values from SearchProperty instead?

Reply:
posted #903 with the alternative I had in mind

Reply:
Not sure if I like representing `boolean` values as strings.... but OK, I will follow your idea.